src/clr-angular/data/tree-view/tree-node.ts

```diff
@@ -77,6 +77,15 @@ export class ClrTreeNode<T> implements OnInit, OnDestroy {
     return !!this.expandService.expandable || this._model.children.length > 0;
   }
 
+  @Input('clrDisabled')
+  get disabled(): boolean {
+    return this._model.disabled.value;
+  }
+  set disabled(value: boolean) {
+    this._model.isDisabledInputSetToTrue = !!value;
+    this._model.setDisabled(!!value, this.featuresService.eager, this.featuresService.eager);
```

Comment:
To make this work properly and avoid a repeat of #3073, we should set a flag prior to running this line and reset it right afterwards. See commit d5ea70da1c0c7aa563ca260195f07b6b199f4efe, and update to the following.

```
set disabled(value: boolean) {
  this._model.isDisabledInputSetToTrue = !!value;
  this.skipEmitChange = true;
  this._model.setDisabled(!!value, this.featuresService.eager, this.featuresService.eager);
  this.skipEmitChange = false;
}
```